xml/System.IO/TextWriter.xml

```diff
@@ -1414,9 +1414,19 @@ This member is an explicit interface member implementation. It can be used only
         <Parameter Name="buffer" Type="System.ReadOnlySpan&lt;System.Char&gt;" Index="0" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="buffer">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="buffer">The character span to write.</param>
+        <summary>Writes a character span to the text stream.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The text representation of the specified value is produced by calling the <xref:System.ReadOnlySpan&lt;System.Char&gt;.ToString%2A?displayProperty=nameWithType> method.
```

Comment:
```suggestion
The text representation of the specified value is produced by calling the [ReadOnlySpan<Char>.ToString](xref:System.ReadOnlySpan%601.ToString%2A) method.
```

Reply:
@rpetrusha oh, I see what you are doing here. I'll edit the malformed one in the first file to look like this one.